api/MediaKeySession.json

```diff
@@ -346,7 +346,7 @@
               "version_added": "55"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
Confluence agrees that `onkeystatuseschange` and `onmessage` was not in Edge 18, FWIW.